docs/sources/release-notes/v2.2.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Grafana Mimir version 2.2 release notes"
+menuTitle: "V2.2 release notes"
+description: "Release notes for Grafana Mimir version 2.2"
+weight: 300
+---
+
+# Grafana Mimir version 2.2 release notes
+
+Grafana Labs is excited to announce version 2.2 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+Highlights include the top features, enhancements, and bugfixes in this release. If you are upgrading from Grafana Mimir 2.1, there is migration-related information as well.
+For the complete list of changes, see the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Support for ingesting out-of-order samples**: Grafana Mimir includes new, experimental support for ingesting out-of-order samples.
+  This support is configurable, with users able to set how far out-of-order Mimir will accept samples on a per-tenant basis.
+  This feature still needs additional testing; we do not recommend using it in production settings yet.
+  For documentation see [Configuring out-of-order samples ingestion]({{< relref "../operators-guide/configuring/configuring-out-of-order-samples-ingestion.md" >}})
+
+- **Improved error messages**: The error messages that Mimir reports are more human readable, and the messages include error codes that are easily searchable.
+  For descriptions of errors, see the [Grafana Mimir runbooks / Errors catalog]({{< relref "../operators-guide/mimir-runbooks/_index.md#errors-catalog" >}}).
+
+- **Configurable prefix for object storage**: Mimir can now store block data, rules, and alerts in one bucket, each under its own user-defined prefix, rather than requiring one bucket for each.
+  You can configure the storage prefix by using `-<storage>.storage-prefix` option for corresponding storage: `ruler-storage`, `alertmanager-storage` or `blocks-storage`.
```

Comment:
```suggestion
- **Configurable prefix for object storage**: Mimir can now store block data, rules, and alerts in one bucket, with each under its own user-defined prefix, rather than requiring one bucket for each.
  You can configure the storage prefix by using `-<storage>.storage-prefix` option for corresponding storage: `ruler-storage`, `alertmanager-storage` or `blocks-storage`.
```